backport.sh

```diff
@@ -118,24 +142,25 @@ fi
 while [ $# -gt 0 ]; do
         case $1 in
                 -c|--create-tree)
+			if [ ! $# -gt 1 ]; then
+				echo "No option provided for creating the tree"
+				exit;
+			fi
                         echo "Creating the tree"
```

Comment:
no option provided means please proceed with full tree( feature ) it should not exit